### DISCUSSION TITLE
How to release a domain I own for github pages?

### DISCUSSION THREAD

**Discussion by jorixyz** [community/community#39536] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi

I unfortunately got "hacked" (litchfieldscoven.com) and someone registered a domain I own in his github pages.

DNS entries are fine and I verified the domain with github using the _github-pages-challenge TXT DNS.

I get the following message when I try to register the domain as a custom domain in the repository holding the github pages: _The custom domain `litchfieldscoven.com` is already taken. If you are the owner of this domain, check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages for information about how to verify and release this domain._  

Now that I validated the domain, how can I force release it so I can register it again as my custom github pages domain?

**Comment by alekseypotapov-dev** [comment]
I'm having same issue. 
With domain `alekseypotapov.dev`.
Idk, whether I got hacked or not. Some sites recommend wait for 7 days until Github will allow park your GH Page on the verified custom domain.

**Comment by jfullstackdev** [comment, marked as answer]
try to wait as recommended above

after that, try Support too, although there is no assurance

https://support.github.com/contact/account
